common-content-dataflow/modules/ROOT/partials/changelog.adoc

```diff
@@ -0,0 +1,101 @@
+= Changelog
+
+This page lists changes to the Dataflow Flex Template for BigQuery and/or Google Cloud to Neo4j.
+
+You can check the deployment status of a release by running `gsutil ls gs://dataflow-templates/`.
+
+Releases are gradually deployed, some regions may be updated before others.
+To check the release status of https://cloud.google.com/compute/docs/regions-zones[a particular region], you can run `gsutil ls gs://dataflow-templates-$\{region}/` (e.g. `gsutil ls gs://dataflow-templates-europe-west2/`).
+
+== Release 2025-06-10-01_RC00
+
+* Add support to see underlying Neo4j driver's Bolt logs
+
+== Release 2025-01-26-00_RC00
+
+* Add support for calendar-based version names of the Neo4j database
```

Comment:
What does this mean? What are some examples of `version names` I would see?

Reply:
Ah, right, I see. I don't know if it's unnecessary information to highlight - if we do highlight it, I would phrase it as "Support for self-managed deployments of Neo4j 2025.x", because that's what's relevant to customers.

Reply:
Thank you so much for the input, I will add this!